### PR TITLE
Fix the local run by separating app config files

### PR DESCRIPTION
**Pull request by redhatHameed**
Currently,  unable to run locally which cause by PR #5  . so added fix for the local run by separating the app config files.

### CODE REVIEW

**Review by sgahlot — APPROVED** [review]

File: app-config.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 app:
-  title: Backstage App
+  title: Scaffolded Backstage App
   baseUrl: http://localhost:3000
 
 organization:
-  name: AppEng-Backstage
+  name: My Company
```
Comment (lokeshrangineni) [inline app-config.yaml]: why not keep the same name??

File: app-config-dev.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    # host: 127.0.0.1
+  csp:
+    connect-src: ["'self'", 'http:', 'https:']
+    # Content-Security-Policy directives follow the Helmet format: https://helmetjs.github.io/#reference
+    # Default Helmet Content-Security-Policy values can be removed by setting the key to false
+  cors:
+    origin: http://localhost:3000
+    methods: [GET, HEAD, PATCH, POST, PUT, DELETE]
+    credentials: true
+  database:
```
Comment (lokeshrangineni) [inline app-config-dev.yaml]: for the dev i prefer in memory database. just my preference. You can check with Sandeep also if he feels diferent.

Comment (redhatHameed) [reply app-config-dev.yaml]: This file is used for OpenShift deployment. I am installing the PostgreSQL database within template. I can rename the file to **app-config-openshift.yaml** if that makes it clearer. For local development, **app-config.yaml** will be used, which is already set up with an in-memory database

File: app-config.yaml (hunk 1 above)
Comment (redhatHameed) [reply app-config.yaml]: I just reverted this file as pervious version. However updated accordingly

File: app-config-dev.yaml (hunk 2 above)
Comment (lokeshrangineni) [reply app-config-dev.yaml]: I think renaming makes much more sense and is clear. Thank you.

**Pull request merged**